Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.11
+
+WORKDIR /app
+
+COPY requirements.txt .
```

Comment:
Ensure that the `requirements.txt` file is present in the build context. Otherwise, the `COPY` command will fail.